frontend/packages/dev-console/src/components/BuildConfigPage.tsx

```diff
@@ -20,7 +20,9 @@ const BuildConfigPage: React.FC<BuildConfigPageProps> = ({ noProjectsAvailable,
         <title>Builds</title>
       </Helmet>
       {namespace ? (
-        <BuildConfigsPage {...props} mock={noProjectsAvailable} />
+        <div className="co-m-list">
```

Comment:
The reason this happens is because margins do not collapse between siblings in a flex layout.

The fix you have is correct. Inserting a `div` between the `#content-scrollable` and the children. However I do not understand why you are adding `co-m-list` here?

Reply:
That seems to be the pattern with all of the list pages in console, so thought I'll use it to be consistent.

Reply:
Perhaps it is. But i cannot see it applying any styling and therefore a change to the class has potential to break us because its use was for a different purpose. Unsure but best to omit for now.

Reply:
Done. Removed the class.